### PR TITLE
[MM-58805]: Remove Standalone code part - 1

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
The PR mainly focus on removing standalone code which include
- isPlugin
- isSingleUser

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -154,6 +154,14 @@ endif
 
 	@echo plugin built at: dist/$(BUNDLE_NAME)
 
+info: ## Display build information
```
Comment (Rajat-Dabade) [inline Makefile:157]: This was only added for debugging purposes. Will remove this after the plugin becomes stable. 
Created the issue to track this down: https://github.com/mattermost/mattermost-plugin-boards/issues/3

**Review by harshilsharma63 — CHANGES_REQUESTED** [review]
Look fine overall but we need to keep those guest user checks as they are still needed in plugin. For example, a guest user in Mattermost shouldn't be allowed to access all the templates.

File: server/api/archive.go [hunk 2]
```diff
@@ -137,16 +137,6 @@ func (a *API) handleArchiveImport(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	isGuest, err := a.userIsGuest(userID)
```
[the remark this answers is not in the document]
Comment (Rajat-Dabade) [reply server/api/archive.go]: Reverted back.

File: server/api/archive.go [hunk 3]
```diff
@@ -220,13 +210,7 @@ func (a *API) handleArchiveExportTeam(w http.ResponseWriter, r *http.Request) {
 	defer a.audit.LogRecord(audit.LevelRead, auditRec)
 	auditRec.AddMeta("TeamID", teamID)
 
-	isGuest, err := a.userIsGuest(userID)
```
[the remark this answers is not in the document]
Comment (Rajat-Dabade) [reply server/api/archive.go]: Done.

**Review by sbishel — APPROVED** [review]
Looks good. I believe all the Cypress tests run in Standalone code. I don't see any updates for those here. So I would expect those to fail.

**Review by marianunez — APPROVED** [review]

**Pull request merged**